js/views/modals/purchase/Purchase.js

```diff
@@ -446,19 +464,18 @@ export default class extends BaseModal {
         })
           .done((data) => {
             this.setState({ phase: 'pending' });
-            this.purchase.set(this.purchase.parse(data));
             this.payment = this.createChild(Payment, {
-              balanceRemaining: this.purchase.get('amount'),
-              paymentAddress: this.purchase.get('paymentAddress'),
-              orderId: this.purchase.get('orderId'),
+              balanceRemaining: integerToDecimal(data.amount, paymentCoin),
+              paymentAddress: data.paymentAddress,
+              orderId: data.orderId,
```

Comment:
The only thing the purchase model was used for was to get the balanceRemaining value, so I refactored it out.